app/mailers/admin_mailer.rb

```diff
@@ -0,0 +1,26 @@
+class AdminMailer < ActionMailer::Base
+  include ContributorHelper
+
+  before_filter :add_inline_attachments!
+
+  default from: "FromThePage <support@fromthepage.com>"
+  layout "admin_mailer"
+  
+  def contributor_stats(collection_id, start_date, end_date)
+
+    new_contributors(collection_id, start_date, end_date)
+
+    mail from: SENDING_EMAIL_ADDRESS, to: ADMIN_EMAILS, subject: "New Transcription Information "
```

Comment:
You probably can - do you want to send it to the collection owner or pass it in the rake task?